Rules

```diff
@@ -19,6 +19,7 @@
 preprocess do
   add_version_attribute
   add_solution_attribute
+  add_button_attribute
```

Comment:
May be simpler to just check @item[:solution] != 'soarian' in the header layout? That way, if we add other solutions, it's obvious when this piece depends on solution?

Reply:
The only issue with that is that the button will show up on the homepage. Not sure if that is expected behavior you would want.

Reply:
eh. probably not. Darn.